### PR TITLE
c

### PR DESCRIPTION
**Pull request by jennyraj**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,8 +33,7 @@
   },
   "dependencies": {
     "angular2": "2.0.0-beta.0",
-    "es6-promise": "^3.0.2",
-    "es6-shim": "^0.34.0",
+    "core-js": "^1.2.6",
```
Comment (gdi2290) [inline package.json]: I switched back to es6-promise and es6-shim so developers won't be confused when they npm install and see that angular2 has a peer dependency of these two

File: src/vendor.ts [hunk 2]
```diff
@@ -1,9 +1,7 @@
 // Polyfills
-import 'es6-promise';
-import 'es6-shim';
+import 'core-js';
 import 'reflect-metadata';
-import 'zone.js/dist/zone-microtask';
-import 'zone.js/dist/long-stack-trace-zone';
+import 'zone.js';
```
Comment (gdi2290) [inline src/vendor.ts]: I updated this to use the source version of these bundles in order to play well with webpack

**Comment by gdi2290** [comment]
can you rebase and make this pull-request only about removing `noParse`

**Comment by jennyraj** [comment]
Sorry, my mistake. Did not mean to create a pull request on Angular class repo

**Comment by jennyraj** [comment]
Sorry, created by mistake

**Comment by gdi2290** [comment]
no problem

**Pull request closed without merging**